service/gcs/bridge/bridge.go

```diff
@@ -470,10 +470,14 @@ func (b *Bridge) getProperties(w ResponseWriter, r *Request) {
 		return
 	}
 
-	propertyJSON, err := json.Marshal(properties)
-	if err != nil {
-		w.Error(request.ActivityID, errors.Wrapf(err, "failed to marshal properties into JSON: %v", properties))
-		return
+	propertyJSON := []byte("{}")
```

Comment:
What does setting this default help us?

Reply:
This prevents the GCS from returning with the JSON string `"null"` if `GetProperties` is called on a container which has been created, but which hasn't been given its first process yet. This way, the HCS receives back an empty properties response, rather than `"null"` and throwing invalid JSON.

Reply:
Oh the lines of context really killed me on that one. I see later how its used.